### DISCUSSION TITLE
Loading times

### DISCUSSION THREAD

**Discussion by srykho22** [kubilus1/autoortho#160] [General]
I'm having issues with AO taking over an hour and a half to start up, with my flights also hanging for over thirty minutes at a time every time it gets new textures. I've gone as far down as minimum ZL 14 and things haven't really changed. Max wait time at 0.25 shows the majority of tiles as just pure green (didn't download), whereas a wait time of 1.50 doesn't load in, even overnight. 

I don't clear the cache, and I have experimented with having the cache on both an external SSD and external HDD. Speedtest says my internet is around 180Mbps download, with task manager saying my ethernet is using 17Mbps with Auto Ortho running. I've also seen the usage go up with other apps when I tested if my ISP was throttling, and I've also tried the new version and reverting to older versions, with barely any difference.

Auto Ortho gives a lot of "empty chunk data, skip" errors, but VPN doesn't really help there. Is there anything I can do?

**Comment by BinaryOutlook** [comment]
Did you setup a Map Type Override? Should be left to blank for faster loading time.

**Reply by srykho22** [reply]
I've tried Bing, Arcgis, and no override. No noticeable difference, though I suppose everything meshes when loading takes that long
